Corona-Warn-App/src/main/res/values-de/notifications_strings.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<resources xmlns:tools="http://schemas.android.com/tools" xmlns:xliff="urn:oasis:names:tc:xliff:document:1.2" tools:ignore="MissingTranslation">
+
+    <!-- ####################################
+                  Common
+    ###################################### -->
+    <!-- XHED: notification management: delta onboarding: header -->
+    <string name="nm_title">"Mitteilungen verwalten"</string>
+    <!-- XTXT: notification management: delta onboarding: point 1 -->
+    <string name="nm_point_overall">"Sie generell Mitteilungen erhalten möchten"</string>
+    <!-- XTXT: notification management: delta onboarding: point 2 -->
+    <string name="nm_point_block_screen">"Mitteilungstexte auf dem Sperrbildschirm angezeigt werden sollen"</string>
+    <!-- XTXT: notification management: delta onboarding: point 3 -->
+    <string name="nm_point_unclock_screen">"Mitteilungstexte erst nach Entsperrung des Smartphones angezeigt werden sollen"</string>
+    <!-- XHED: notification management: delta onboarding: faq -->
+    <string name="nm_faq">"Details zu den Einstellungen Ihres jeweiligen Betriebssystems finden Sie in den FAQ."</string>
+    <!-- XTXT: notification management: delta onboarding: faq label -->
+    <string name="nm_faq_label">"FAQ"</string>
+    <!-- XTXT: notification management: delta onboarding: faq link -->
+    <string name="nm_faq_link">"https://www.coronawarn.app/en/faq/#notification_settings"</string>
+
+    <!-- ####################################
+                  Delta Onboarding
+    ###################################### -->
+
+    <!-- XTXT: notification management: delta onboarding: body -->
+    <string name="nm_delta_onboarding_body">"Die Einstellungen für Mitteilungen werden ab jetzt einheitlich in den Geräteeinstellungen Ihres Smartphones verwaltet."</string>
+    <!-- XTXT: notification management: delta onboarding: subtitle -->
+    <string name="nm_delta_onboarding_subtitle">"Je nach Betriebssystem-Version und Gerätemodell können Sie festlegen, ob:"</string>
+    <!-- XBUT: notification management: delta onboarding: device settings button -->
+    <string name="nm_delta_onboarding_device_settings_button">"Geräteeinstellungen öffnen"</string>
```

Comment:
@AlexanderAlferov Are the buttons in Android not always capitalized? GERÄTEEINSTELLUNGEN ÖFFNEN ? Sorry, I am not certain, I just saw that before.

Reply:
@InkenTownsend , as we are using material design ([link](https://material.io/components/buttons#anatomy)), all buttons are capitalised. And you also do not need to worry about capitalisation in strings files, as it is done automatically. E.g. you can check the screenshot attached to this PR.

Reply:
👍 @AlexanderAlferov Thanks